_beats/dev-tools/packaging/packages.yml

```diff
@@ -157,6 +157,22 @@ shared:
         template: '{{ elastic_beats_dir }}/dev-tools/packaging/templates/windows/uninstall-service.ps1.tmpl'
         mode: 0755
 
+  - &docker_spec
+    <<: *binary_spec
+    extra_vars:
+      from: 'centos:7'
+      user: '{{ .BeatName }}'
+      linux_capabilities: ''
+    files:
+      '{{.BeatName}}.yml':
+        source: '{{.BeatName}}.docker.yml'
+        mode: 0600
+        config: true
+
+  - &elastic_docker_spec
+    extra_vars:
+      repository: 'docker.elastic.co/beats'
```

Comment:
does this play well with us?

Reply:
yeah, we override this here: https://github.com/elastic/apm-server/pull/1632/files#diff-3db01cb68c42f27f6422c65fa1175e23R193